JS01-04.js

```diff
@@ -0,0 +1,29 @@
+let Arrays = [

```

Comment:
2点気になりました

1. 再代入をしないので `const` でいいと思いました
1. 変数名は慣習として `camelCase` なのと、名前そのものの抽象度がやや高い気がしたので、もう少し具体的な名前を付けた方がよいかと思います

```suggestion
const taskArrays = [
```